generators/entity-server/templates/src/test/gatling/user-files/simulations/EntityGatlingTest.scala.ejs

```diff
@@ -103,8 +103,15 @@ class <%= entityClass %>GatlingTest extends Simulation {
         .exec(http("First unauthenticated request")
         .get("/api/account")
         .headers(headers_http)
-        .check(status.is(401))<% if (authenticationType === 'session' || authenticationType === 'oauth2') { %>
-        .check(headerRegex("Set-Cookie", "XSRF-TOKEN=(.*);[\\s]").saveAs("xsrf_token"))<% } %>).exitHereIfFailed
+        <%_ if ((applicationType === 'microservice' || applicationType === 'gateway') && authenticationType === 'oauth2') { _%>
+        .check(status.is(302))
```

Comment:
with microservices/gateway, it's 302
with monolithic, it's 401

Yeah strange but it works like this. Didn't have time to investigate why